README.md

```diff
@@ -131,7 +131,7 @@ Make sure you've read the [contribution guidelines](http://cakebuild.net/contrib
 
 ## License
 
-Copyright © .NET Foundation, Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman and contributors.
+Copyright © .NET Foundation, Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors.
```

Comment:
@mholo65 Yeah, I am not sure on that one...

http://www.grammarbook.com/punctuation/commas.asp

Thoughts?